webui/react/src/types.ts

```diff
@@ -317,24 +317,31 @@ export interface ExperimentPagination extends WithPagination {
   experiments: ExperimentItem[];
 }
 
-export enum RunState {
-  Active = 'ACTIVE',
-  Paused = 'PAUSED',
-  StoppingCanceled = 'STOPPING_CANCELED',
-  Canceled = 'CANCELED',
-  StoppingCompleted = 'STOPPING_COMPLETED',
-  Completed = 'COMPLETED',
-  StoppingError = 'STOPPING_ERROR',
-  Errored = 'ERROR',
-  Deleted = 'DELETED',
-  Deleting = 'DELETING',
-  DeleteFailed = 'DELETE_FAILED',
-  Unspecified = 'UNSPECIFIED',
-  Queued = 'QUEUED',
-  Pulling = 'PULLING',
-  Starting = 'STARTING',
-  Running = 'RUNNING',
-}
+export const RunState = {
+  Active: 'ACTIVE',
+  Canceled: 'CANCELED',
+  Completed: 'COMPLETED',
+  Deleted: 'DELETED',
+  DeleteFailed: 'DELETE_FAILED',
+  Deleting: 'DELETING',
+  Error: 'ERROR',
+  Paused: 'PAUSED',
+  Pulling: 'PULLING',
+  Queued: 'QUEUED',
+  Running: 'RUNNING',
+  Starting: 'STARTING',
+  StoppingCanceled: 'STOPPING_CANCELED',
+  StoppingCompleted: 'STOPPING_COMPLETED',
+  StoppingError: 'STOPPING_ERROR',
+  StoppingKilled: 'STOPPING_KILLED',
+  Unspecified: 'UNSPECIFIED',
+} as const;
+
+export type RunState = typeof RunState;
+
+export type RunStateKey = keyof RunState;
+
+export type RunStateValue = typeof RunState[RunStateKey];
```

Comment:
You can just name this `RunState`. `'RUNNING'` is a "run state" and the type you're currently using that name for isn't particularly useful if a bit of code needs `typeof RunState` then it can just do `typeof RunState`.